### ISSUE TITLE
TLDR fails on freebsd 13 / aarch 64 

### ISSUE THREAD

**Issue by network-shark** [tldr-pages/tldr-python-client#155]
Hello , 

I tried to use tldr on my raspberry pi , but it seems the platform it not supported yet ? 

```
root@FreeBSD-PI:/home/freebsd # tldr tar
Traceback (most recent call last):
  File "/usr/local/bin/tldr", line 8, in <module>
    sys.exit(cli())
  File "/usr/local/lib/python3.7/site-packages/tldr.py", line 473, in cli
    main()
  File "/usr/local/lib/python3.7/site-packages/tldr.py", line 429, in main
    ).completer = argcomplete.completers.ChoicesCompleter(get_commands())
  File "/usr/local/lib/python3.7/site-packages/tldr.py", line 271, in get_commands
    platforms = get_platform_list()
  File "/usr/local/lib/python3.7/site-packages/tldr.py", line 180, in get_platform_list
    platforms.remove(current_platform)
ValueError: list.remove(x): x not in list
```

```
root@FreeBSD-PI:/home/freebsd # uname -a
FreeBSD FreeBSD-PI 13.0-RC4 FreeBSD 13.0-RC4 #0 releng/13.0-n244719-2e70fcbb154: Mon Mar 29 06:13:04 UTC 2021     root@releng1.nyi.freebsd.org:/usr/obj/usr/src/arm64.aarch64/sys/GENERIC  arm64
```

```
root@FreeBSD-PI:/home/freebsd # python3.7
Python 3.7.9 (default, Jan 21 2021, 01:59:15)
[Clang 11.0.1 (git@github.com:llvm/llvm-project.git llvmorg-11.0.1-0-g43ff75f2c on freebsd13
Type "help", "copyright", "credits" or "license" for more information.
```

**Comment by MasterOdin** [comment]
Could you provide the output of `python3 -c "import sys; print(sys.platform);"`? Just curious, though the fallback behavior for the client should probably be to just assume some sort of linux.

**Comment by MasterOdin** [comment]
Fixed in 6aa95117a8adce2abb423546e05b169f1b69da52.
